docs/core/tools/dotnet-new-list.md

```diff
@@ -22,6 +22,23 @@ dotnet new [<TEMPLATE_NAME>] -l|--list [--author <AUTHOR>] [-lang|--language {"C
 
 The `dotnet new --list` option lists available templates to use with `dotnet new`. If the <TEMPLATE_NAME> is specified, lists templates containing the specified name. This option lists only default and installed templates. To find templates in NuGet that you can install locally, use the [`--search`](dotnet-new-search.md) option.
 
+<!-- markdownlint-disable MD012 -->
+[!INCLUDE [new syntax](../../../includes/dotnet-new-7-0-syntax.md)]
+
+Examples:
+
+- Show help for list subcommand
+
+```
+dotnet new list --help
```

Comment:
can you update [this example](https://github.com/dotnet/docs/blob/9afd7ac45aa807749e4bc07938c3c2b1e13f0c35/docs/core/tools/dotnet-new-sdk-templates.md?plain=1#L13) on the sdk templates page as well?

Reply:
We decided not to update the other articles yet. This command will still work.
I haven't added a note to this article however, do you think we should add it here as well?

Reply:
I think it would be good to add it here as well.